### PR TITLE
Refactoring run scripts and moving paths to new project

### PR DESCRIPTION
**Pull request by nthain**
This PR refactors the paths for the run scripts and points them to the path for the new cloud project (please refer to the e-mail I sent and respond there if there are any issues or concerns). It also reduces the default batch size of the models so that you don't by default run into OOM error when training on ML engine. 

In a future PR, I will refactor so that we will finally have one run script that can run all of the models instead of having a separate run script per model.

### CODE REVIEW

File: experiments/tf_trainer/tf_char_cnn/run.hyperparameter.sh [hunk 1]
```diff
@@ -1,48 +1,10 @@
 #!/bin/bash
 
-GCS_RESOURCES="gs://kaggle-model-experiments/resources"
+source "tf_trainer/common/dataset_config.sh"
 DATETIME=`date '+%Y%m%d_%H%M%S'`
```
Comment (sorensenjs) [inline experiments/tf_trainer/tf_char_cnn/run.hyperparameter.sh]: driveby - use $() instead of backtics.

File: experiments/tf_trainer/common/dataset_config.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    label_dtypes="float"
+    train_steps=24000
+    eval_period=800
+    eval_steps=50
+
+elif [ "$1" == "many_communities" ]; then
+    train_path="${GCS_RESOURCES}/transfer_learning_data/many_communities/20181105_train.tfrecord"
+    valid_path="${GCS_RESOURCES}/transfer_learning_data/many_communities/20181105_validate.tfrecord"
+    labels="removed"
+    label_dtypes="int"
```
Comment (iislucas) [inline experiments/tf_trainer/common/dataset_config.sh]: Removed is an integer? If so maybe explain what the interpretation of the integer is, e.g. 0 = not removed, 1 = removed.

File: experiments/tf_trainer/common/dataset_config.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    train_steps=100000
+    eval_period=800
+    eval_steps=50
+
+else
+    echo "First positional arg must be one of civil_comments, toxicity, many_communities."
+    exit 1
+fi
```
[the remark this answers is not in the document]
Comment (nthain) [reply experiments/tf_trainer/common/dataset_config.sh]: Done.

File: experiments/tf_trainer/common/tfrecord_input.py [hunk 4]
```diff
@@ -27,7 +27,7 @@
                            'Name of feature containing text input.')
 tf.app.flags.DEFINE_boolean('round_labels', True,
                             'Round label features to 0 or 1 if true.')
-tf.app.flags.DEFINE_integer('batch_size', 256,
+tf.app.flags.DEFINE_integer('batch_size', 32,
```
[the remark this answers is not in the document]
Comment (nthain) [reply experiments/tf_trainer/common/tfrecord_input.py]: Removed from here and I'll propose it in a different PR.

File: experiments/tf_trainer/tf_char_cnn/run.hyperparameter.sh (hunk 1 above)
Comment (nthain) [reply experiments/tf_trainer/tf_char_cnn/run.hyperparameter.sh]: Changed all occurrences.

File: experiments/tf_trainer/common/dataset_config.sh (hunk 2 above)
Comment (nthain) [reply experiments/tf_trainer/common/dataset_config.sh]: Done.
